### PR TITLE
Removing sample.bb, replaced with some basic information about how to…

### PR DESCRIPTION
**Pull request by darobs**
… build using bitbake on Yocto.

Added some Intel Edison specific instruction.

### CODE REVIEW

File: samples/simulated_device_cloud_upload/README.md [hunk 1]
```diff
@@ -59,6 +59,36 @@ Windows
...
+The Edison does not support all of the Azure recipes available in the "meta-iot-cloud" repository. To get the Edison image to build, we have to limit the recipes included in the image.
+
+- Remove the azure-iot-gateway-sdk-java target.
+
+    The dependencies for Java require libraries not included in the Edison BSP 
+    source package. To avoid building the java target, add the following line 
+    to your `conf/local.conf` file.
+
+```
+PACKAGECONFIG_pn-azure-iot-gateway-sdk = "bluetooth"
```
Comment (damonbarry) [inline samples/simulated_device_cloud_upload/README.md]: I don't see how adding something about bluetooth to a conf file will prevent the java target from being built?

File: samples/simulated_device_cloud_upload/README.md [hunk 2]
```diff
@@ -59,6 +59,36 @@ Windows
...
+```
+
+- Add the azure-iot-gateway-sdk recipe to the target.
+
+    Edit the Edison image receipe to include the "azure-iot-gateway-sdk" target.  For example:
```
Comment (damonbarry) [inline samples/simulated_device_cloud_upload/README.md]: typo: receipe --> recipe

File: samples/simulated_device_cloud_upload/src/azure-iot-field-gateway-sdk.bb [hunk 3]
```diff
@@ -1,32 +0,0 @@
-DESCRIPTION = ""
```
Comment (damonbarry) [inline samples/simulated_device_cloud_upload/src/azure-iot-field-gateway-sdk.bb]: Yay! Look at all the red!

File: samples/simulated_device_cloud_upload/README.md (hunk 1 above)
Comment (darobs) [inline samples/simulated_device_cloud_upload/README.md]: It's not obvious, sorry.  There are 3 packages in the azure-iot-gateway-sdk recipe: the main package, azure-iot-gateway-sdk-java and azure-iot-gateway-sdk-bluetooth.  This statement will limit the packages to the main and bluetooth packages.  I'll try to make this more clear.

**Pull request merged**